spec.html

```diff
@@ -76,3 +80,25 @@ <h1>String.prototype.replaceAll ( _searchValue_, _replaceValue_ )</h1>
     1. Return _result_.
   </emu-alg>
 </emu-clause>
+
+<emu-clause id="sec-string.prototype.matchall">
+  <h1>String.prototype.matchAll ( _regexp_ )</h1>
+  <p>Performs a regular expression match of the String representing the *this* value against _regexp_ and returns an iterator. Each iteration result's value is an Array object containing the results of the match, or *null* if the String did not match.</p>
+  <p>When the `matchAll` method is called, the following steps are taken:</p>
+  <emu-alg>
+    1. Let _O_ be ? RequireObjectCoercible(*this* value).
+    1. If _regexp_ is neither *undefined* nor *null*, then
+      1. <ins>Let _isRegExp_ be ? IsRegExp(_regexp_).</ins>
+      1. <ins>If _isRegExp_ is true, then</ins>
+        1. <ins>Let _flags_ be ? ToString(? Get(_regexp_, `"flags"`)).</ins>
```

Comment:
If flags is undefined, this should be the empty string, not the string undefined.